src/utils/getFilteredPeople.ts

```diff
@@ -0,0 +1,50 @@
+import { Person } from '../types';
+
+export const getFilteredPeople = (
+  people: Person[],
+  query: string | null,
+  sex: string | null,
+  centuries: string[],
+) => {
+  let filteredPeople = [...people];
```

Comment:
Method `filter` returns a new array every time, so you don’t need to copy it with the spread operator.